homeassistant/components/switchbot/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "switchbot",
   "name": "SwitchBot",
   "documentation": "https://www.home-assistant.io/integrations/switchbot",
-  "requirements": ["PySwitchbot==0.13.0"],
+  "requirements": ["PySwitchbot==0.13.1"],
```

Comment:
0.13.1 contains a print statement. Is that intended or a leftover during debugging?

https://github.com/Danielhiversen/pySwitchbot/blob/db75e9fb302f913f8cb7c86580aa2ab214c39d5e/switchbot/__init__.py#L413

Reply:
Hi @MartinHjelmare,

It was left over during debugging. I'll submit a pull request on the api to remove it.

Had to test so many different combinations and print just made it easier.

Reply:
Hi @MartinHjelmare,

The print statement was removed from upstream api.

Thanks.